app/screens/Settings/SettingsScreen.tsx

```diff
@@ -4,6 +4,7 @@ import {
   TouchableWithoutFeedback,
   ScrollView,
   Image,
+  Platform,
```

Comment:
We have some helper for platform : 
import { t, platform } from "../../utils";

Reply:
it has just two booleans of isAndroid & isIOS and it doesn't help this case. in this case we want to have Items that are os specific and later check that against it. so if you don't mind I won't apply this change